runtime/commonMain/src/kotlinx/serialization/Migrations.kt

```diff
@@ -141,28 +141,31 @@ public inline fun <reified T : Any> StringFormat.stringify(objects: List<T>): St
     replaceWith = ReplaceWith("stringify(map)")
 )
 public inline fun <reified K : Any, reified V : Any> StringFormat.stringify(map: Map<K, V>): String =
-    encodeToString(MapSerializer(serializersModule.getContextualOrDefault<K>(), serializersModule.getContextualOrDefault<V>()), map)
+    encodeToString(MapSerializer(serializersModule.serializer<K>(), serializersModule.serializer<V>()), map)
 
-@ImplicitReflectionSerializer
-@OptIn(UnsafeSerializationApi::class)
 @Deprecated(
-    level = DeprecationLevel.WARNING,
+    level = DeprecationLevel.ERROR,
     message = "This method is deprecated for removal. Please use reified getContextualOrDefault<T>() instead",
     replaceWith = ReplaceWith("getContextual(klass) ?: klass.serializer()")
 )
 public fun <T : Any> SerializersModule.getContextualOrDefault(klass: KClass<T>): KSerializer<T> =
     getContextual(klass) ?: klass.serializer()
 
-@ImplicitReflectionSerializer
-@OptIn(UnsafeSerializationApi::class)
 @Deprecated(
-    level = DeprecationLevel.WARNING,
+    level = DeprecationLevel.ERROR,
     message = "This method is deprecated for removal. Please use reified getContextualOrDefault<T>() instead",
-    replaceWith = ReplaceWith("getContextualOrDefault<T>()")
+    replaceWith = ReplaceWith("serializer<T>()")
 )
 public fun <T : Any> SerializersModule.getContextualOrDefault(value: T): KSerializer<T> =
     getContextual(value::class)?.cast() ?: value::class.serializer().cast()
 
+@Deprecated(
+    level = DeprecationLevel.ERROR,
+    message = "This method is deprecated for removal. Please use reified getContextualOrDefault<T>() instead",
```

Comment:
```suggestion
    message = "This method is deprecated for removal. Please use reified serializer<T>() instead",
```

But since this method wasn't released yet, maybe it makes sense not to include migration?

Reply:
Good idea